teamcity: failed tests on master: acceptance/TestGossipRestart

The following tests appear to have failed:

[#605078](https://teamcity.cockroachdb.com/viewLog.html?buildId=605078):


```
--- FAIL: acceptance/TestGossipRestart (18.240s)
test_log_scope.go:81: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestGossipRestart448684357
test_log_scope.go:62: use -show-logs to present logs inline
--- FAIL: acceptance/TestGossipRestart: TestGossipRestart/runMode=local (18.230s)
gossip_peerings_test.go:171: pq: database "test" already exists
------- Stdout: -------
CockroachDB node starting at 2018-04-15 19:39:43.439385203 +0000 UTC (took 0.6s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:45893
sql:                 postgresql://root@127.0.0.1:33595?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/1
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/1/cockroach-temp671799869
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/1/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/1
status:              initialized new cluster
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              1
CockroachDB node starting at 2018-04-15 19:39:43.868953945 +0000 UTC (took 0.2s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:45745
sql:                 postgresql://root@127.0.0.1:42725?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/3
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/3/cockroach-temp581812759
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/3/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/3
status:              initialized new node, joined pre-existing cluster
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              2
CockroachDB node starting at 2018-04-15 19:39:43.877134284 +0000 UTC (took 0.2s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:37641
sql:                 postgresql://root@127.0.0.1:45767?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/2
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/2/cockroach-temp653449571
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/2/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/2
status:              initialized new node, joined pre-existing cluster
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              3
CockroachDB node starting at 2018-04-15 19:39:43.890533562 +0000 UTC (took 0.2s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:35549
sql:                 postgresql://root@127.0.0.1:37537?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/4
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/4/cockroach-temp435279612
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/4/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/4
status:              initialized new node, joined pre-existing cluster
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              4
CockroachDB node starting at 2018-04-15 19:39:50.18192255 +0000 UTC (took 4.3s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:43715
sql:                 postgresql://root@127.0.0.1:37493?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/1
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/1/cockroach-temp248720633
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/1/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/1
status:              restarted pre-existing node
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              1
CockroachDB node starting at 2018-04-15 19:39:50.320254703 +0000 UTC (took 4.2s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:36833
sql:                 postgresql://root@127.0.0.1:43185?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/4
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/4/cockroach-temp277994779
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/4/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/4
status:              restarted pre-existing node
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              4
CockroachDB node starting at 2018-04-15 19:39:50.373097788 +0000 UTC (took 4.2s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:38615
sql:                 postgresql://root@127.0.0.1:36301?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/3
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/3/cockroach-temp763634989
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/3/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/3
status:              restarted pre-existing node
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              2
CockroachDB node starting at 2018-04-15 19:39:50.380967613 +0000 UTC (took 4.3s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:34553
sql:                 postgresql://root@127.0.0.1:42789?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/2
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/2/cockroach-temp314050945
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/2/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/2
status:              restarted pre-existing node
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              3
CockroachDB node starting at 2018-04-15 19:39:55.550612855 +0000 UTC (took 4.5s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:37109
sql:                 postgresql://root@127.0.0.1:37383?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/4
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/4/cockroach-temp944931254
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/4/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/4
status:              restarted pre-existing node
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              4
CockroachDB node starting at 2018-04-15 19:39:55.579796953 +0000 UTC (took 4.5s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:45913
sql:                 postgresql://root@127.0.0.1:40891?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/2
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/2/cockroach-temp935387728
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/2/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/2
status:              restarted pre-existing node
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              3
CockroachDB node starting at 2018-04-15 19:39:55.613154309 +0000 UTC (took 4.7s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:33549
sql:                 postgresql://root@127.0.0.1:34989?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/1
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/1/cockroach-temp491798889
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/1/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/1
status:              restarted pre-existing node
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              1
CockroachDB node starting at 2018-04-15 19:40:00.755516885 +0000 UTC (took 9.7s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:38057
sql:                 postgresql://root@127.0.0.1:44045?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/3
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/3/cockroach-temp447647501
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/3/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/3
status:              restarted pre-existing node
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              2
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestGossipRestart448684357
--- FAIL: acceptance/TestGossipRestart (18.240s)
test_log_scope.go:81: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestGossipRestart448684357
test_log_scope.go:62: use -show-logs to present logs inline
--- FAIL: acceptance/TestGossipRestart: TestGossipRestart/runMode=local (18.230s)
gossip_peerings_test.go:171: pq: database "test" already exists
------- Stdout: -------
CockroachDB node starting at 2018-04-15 19:39:43.439385203 +0000 UTC (took 0.6s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:45893
sql:                 postgresql://root@127.0.0.1:33595?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/1
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/1/cockroach-temp671799869
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/1/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/1
status:              initialized new cluster
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              1
CockroachDB node starting at 2018-04-15 19:39:43.868953945 +0000 UTC (took 0.2s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:45745
sql:                 postgresql://root@127.0.0.1:42725?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/3
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/3/cockroach-temp581812759
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/3/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/3
status:              initialized new node, joined pre-existing cluster
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              2
CockroachDB node starting at 2018-04-15 19:39:43.877134284 +0000 UTC (took 0.2s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:37641
sql:                 postgresql://root@127.0.0.1:45767?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/2
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/2/cockroach-temp653449571
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/2/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/2
status:              initialized new node, joined pre-existing cluster
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              3
CockroachDB node starting at 2018-04-15 19:39:43.890533562 +0000 UTC (took 0.2s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:35549
sql:                 postgresql://root@127.0.0.1:37537?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/4
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/4/cockroach-temp435279612
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/4/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/4
status:              initialized new node, joined pre-existing cluster
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              4
CockroachDB node starting at 2018-04-15 19:39:50.18192255 +0000 UTC (took 4.3s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:43715
sql:                 postgresql://root@127.0.0.1:37493?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/1
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/1/cockroach-temp248720633
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/1/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/1
status:              restarted pre-existing node
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              1
CockroachDB node starting at 2018-04-15 19:39:50.320254703 +0000 UTC (took 4.2s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:36833
sql:                 postgresql://root@127.0.0.1:43185?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/4
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/4/cockroach-temp277994779
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/4/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/4
status:              restarted pre-existing node
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              4
CockroachDB node starting at 2018-04-15 19:39:50.373097788 +0000 UTC (took 4.2s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:38615
sql:                 postgresql://root@127.0.0.1:36301?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/3
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/3/cockroach-temp763634989
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/3/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/3
status:              restarted pre-existing node
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              2
CockroachDB node starting at 2018-04-15 19:39:50.380967613 +0000 UTC (took 4.3s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:34553
sql:                 postgresql://root@127.0.0.1:42789?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/2
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/2/cockroach-temp314050945
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/2/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/2
status:              restarted pre-existing node
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              3
CockroachDB node starting at 2018-04-15 19:39:55.550612855 +0000 UTC (took 4.5s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:37109
sql:                 postgresql://root@127.0.0.1:37383?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/4
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/4/cockroach-temp944931254
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/4/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/4
status:              restarted pre-existing node
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              4
CockroachDB node starting at 2018-04-15 19:39:55.579796953 +0000 UTC (took 4.5s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:45913
sql:                 postgresql://root@127.0.0.1:40891?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/2
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/2/cockroach-temp935387728
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/2/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/2
status:              restarted pre-existing node
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              3
CockroachDB node starting at 2018-04-15 19:39:55.613154309 +0000 UTC (took 4.7s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:33549
sql:                 postgresql://root@127.0.0.1:34989?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/1
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/1/cockroach-temp491798889
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/1/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/1
status:              restarted pre-existing node
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              1
CockroachDB node starting at 2018-04-15 19:40:00.755516885 +0000 UTC (took 9.7s)
build:               CCL v2.1.0-alpha.20180416-127-g33005f1 @ 2018/04/15 19:27:15 (go1.10)
admin:               http://127.0.0.1:38057
sql:                 postgresql://root@127.0.0.1:44045?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestart/runMode=local/3
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/3/cockroach-temp447647501
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/3/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster444106720/3
status:              restarted pre-existing node
clusterID:           25c31d41-903b-4aca-809b-699368c7d4cf
nodeID:              2
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestGossipRestart448684357

```

Please assign, take a look and update the issue accordingly.

--- a/pkg/acceptance/gossip_peerings_test.go
+++ b/pkg/acceptance/gossip_peerings_test.go
@@ -167,6 +167,9 @@ func testClusterConnectedAndFunctional(ctx context.Context, t *testing.T, c clus
 			t.Fatal(err)
 		}
 		if i == 0 {
+			if _, err := db.Exec("DROP DATABASE IF EXISTS test"); err != nil {
+				t.Fatal(err)
+			}
 			if _, err := db.Exec("CREATE DATABASE test"); err != nil {
 				t.Fatal(err)
 			}

